vertical-pod-autoscaler/e2e/v1/updater.go

```diff
@@ -207,6 +208,70 @@ var _ = UpdaterE2eDescribe("Updater", func() {
 	})
 })
 
+var _ = UpdaterE2eDescribe("Updater with PerVPAConfig", func() {
+	const replicas = 3
+	const statusUpdateInterval = 10 * time.Second
+	f := framework.NewDefaultFramework("vertical-pod-autoscaling")
+	f.NamespacePodSecurityEnforceLevel = podsecurity.LevelBaseline
+
+	f.It("does not evict pods with OOM when threshold is very small", framework.WithFeatureGate(features.PerVPAConfig), func() {
+		ginkgo.By("Setting up the Admission Controller status")
+		stopCh := make(chan struct{})
+		statusUpdater := status.NewUpdater(
+			f.ClientSet,
+			status.AdmissionControllerStatusName,
+			status.AdmissionControllerStatusNamespace,
+			statusUpdateInterval,
+			"e2e test",
+		)
+		defer func() {
+			ginkgo.By("Deleting the Admission Controller status")
+			close(stopCh)
+			err := f.ClientSet.CoordinationV1().Leases(status.AdmissionControllerStatusNamespace).
+				Delete(context.TODO(), status.AdmissionControllerStatusName, metav1.DeleteOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+		statusUpdater.Run(stopCh)
+
+		ginkgo.By("Setting up a deployment that will OOM")
+		runOomingReplicationController(
+			f.ClientSet,
+			f.Namespace.Name,
+			"hamster",
+			replicas,
+		)
+
+		ginkgo.By("Waiting for pods to be created and OOM")
+		time.Sleep(10 * time.Second)
+
+		podList, err := GetOOMPods(f)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(len(podList.Items)).To(gomega.BeNumerically(">", 0))
+
+		disabledThreshold := 0 * time.Second // Disable quick OOM eviction
```

Comment:
with [this commit](https://github.com/kubernetes/autoscaler/pull/8692/commits/46b0ed29d199d5f9a369b4b1ffcc621a486d3d2f), this test is not going to fail?